pgjdbc/src/test/java/org/postgresql/test/jdbc2/PGPropertyTest.java

```diff
@@ -150,6 +152,18 @@ public void testDataSourceProperties() throws Exception {
     }
   }
 
+  /**
+   * Test to make sure that setURL doesn't overwrite autosave
+   * more should be put in but this scratches the current itch
+   */
+  @Test
+  public void testOverWriteDSProperties() throws Exception {
+    PGSimpleDataSource dataSource = new PGSimpleDataSource();
+    dataSource.setAutosave(AutoSave.CONSERVATIVE);
+    dataSource.setURL("jdbc:postgresql://localhost:5432/postgres");
```

Comment:
@davecramer , is `dataSource.setURL("jdbc:postgresql://localhost:5432/postgres?autoSave=never");` supported?